docs/python_docs/python/tutorials/getting-started/crash-course/1-ndarray.md

```diff
@@ -15,113 +15,108 @@
 <!--- specific language governing permissions and limitations -->
 <!--- under the License. -->
 
-# Manipulate data with `ndarray`
+# Step 1: Manipulate data with NP on MXNet
 
-We'll start by introducing the `NDArray`, MXNet’s primary tool for storing and transforming data. If you’ve worked with `NumPy` before, you’ll notice that an NDArray is, by design, similar to NumPy’s multi-dimensional array.
+This getting started exercise introduces the `np` package, which is the primary tool for storing and
```

Comment:
Let's summarize the extent to which `np` is similar and link to a document containing details ` docs/python_docs/python/tutorials/getting-started/deepnumpy/deepnumpy-vs-numpy.md`

Reply:
Done